README.md

```diff
@@ -221,7 +221,7 @@ functions.
 <!-- `> bash ./supported-programs.sh` -->
 
 <!-- BEGIN mdsh -->
-`treefmt-nix` currently supports 106 formatters:
+`treefmt-nix` currently supports 108 formatters:
```

Comment:
The number should be correct if this PR is merged after https://github.com/numtide/treefmt-nix/pull/360.